src/Command/Project/ProjectCreateCommand.php

```diff
@@ -247,23 +317,85 @@ protected function getFields()
             },
             'default' => in_array('development', $this->getAvailablePlans()) ? 'development' : null,
             'allowOther' => true,
-          ]),
-          'environments' => new Field('Environments', [
+          ]);
+          $fields['environments'] = new Field('Environments', [
             'optionName' => 'environments',
             'description' => 'The number of environments',
             'default' => 3,
             'validator' => function ($value) {
                 return is_numeric($value) && $value > 0 && $value < 50;
             },
-          ]),
-          'storage' => new Field('Storage', [
+          ]);
+          $fields['storage'] = new Field('Storage', [
             'description' => 'The amount of storage per environment, in GiB',
             'default' => 5,
             'validator' => function ($value) {
                 return is_numeric($value) && $value > 0 && $value < 1024;
             },
-          ]),
-        ];
+          ]);
+        return $fields;
+    }
+
+     /**
+     * Returns a list of ConsoleForm form fields for this command.
+     *
+     * @return Field[]
+     */
+    protected function getTemplateFields($url_provided)
+    {
+        $fields = [];
+        
+        if (!$url_provided) {
+            $fields['template_option'] = new OptionsField('Template Options', [
+                'optionName' => 'template_option',
+                'options' => [
+                    'Provide your own template url.',
+                    'Choose a template from the catalog.',
+                    'No template at this time.',
+                ],
+                'description' => 'Choose a template, provide a url or choose not to use one.',
+                'includeAsOption' => false,
+            ]);
+    
+            $fields['catalog_url'] = new OptionsField('Catalog', [
```

Comment:
It's not a "catalog URL", it's a template URL... this needs to be a bit clearer, even

```suggestion
            $fields['template_url_from_catalog'] = new OptionsField('Template (from a catalog)', [
```

Reply:
update made